src/Jackett.Common/Indexers/ApiBay.cs

```diff
@@ -0,0 +1,225 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Jackett.Common.Converters;
+using Jackett.Common.Models;
+using Jackett.Common.Models.IndexerConfig;
+using Jackett.Common.Services.Interfaces;
+using Jackett.Common.Utils;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NLog;
+using WebClient = Jackett.Common.Utils.Clients.WebClient;
+
+namespace Jackett.Common.Indexers
+{
+    /// <summary>
+    /// This <see cref="BaseWebIndexer"/> implementation is for the The Pirate Bay API (https://apibay.org).
+    /// </summary>
+    public class ApiBay : BaseWebIndexer
+    {
+        private const string KeyInfoHash = "{info_hash}";
+
+        private static readonly string MagnetUri =
+            $"magnet:?xt=urn:btih:{KeyInfoHash}&dn=Carey%20M.%20Tribe%20of%20Hackers." +
+            "%20Cybersecurity%20Advice...2019&tr=udp%3A%2F%2Ftracker.coppersurfer.tk" +
+            "%3A6969%2Fannounce&tr=udp%3A%2F%2F9.rarbg.to%3A2920%2Fannounce&tr=udp%3" +
+            "A%2F%2Ftracker.opentrackr.org%3A1337&tr=udp%3A%2F%2Ftracker.internetwar" +
+            "riors.net%3A1337%2Fannounce&tr=udp%3A%2F%2Ftracker.leechers-paradise.or" +
+            "g%3A6969%2Fannounce&tr=udp%3A%2F%2Ftracker.coppersurfer.tk%3A6969%2Fann" +
+            "ounce&tr=udp%3A%2F%2Ftracker.pirateparty.gr%3A6969%2Fannounce&tr=udp%3A" +
+            "%2F%2Ftracker.cyberia.is%3A6969%2Fannounce";
```

Comment:
```suggestion
        private static readonly string MagnetUri =
            $"magnet:?xt=urn:btih:{KeyInfoHash}&tr=udp%3A%2F%2Ftracker.coppersurfer.tk" +
            "%3A6969%2Fannounce&tr=udp%3A%2F%2F9.rarbg.to%3A2920%2Fannounce&tr=udp%3" +
            "A%2F%2Ftracker.opentrackr.org%3A1337&tr=udp%3A%2F%2Ftracker.internetwar" +
            "riors.net%3A1337%2Fannounce&tr=udp%3A%2F%2Ftracker.leechers-paradise.or" +
            "g%3A6969%2Fannounce&tr=udp%3A%2F%2Ftracker.coppersurfer.tk%3A6969%2Fann" +
            "ounce&tr=udp%3A%2F%2Ftracker.pirateparty.gr%3A6969%2Fannounce&tr=udp%3A" +
            "%2F%2Ftracker.cyberia.is%3A6969%2Fannounce";
```
Otherwise it names all magnet downloads `Carey M.​ Tribe of Hackers.​ Cybersecurity Advice.​.​.​2019`

Reply:
Nice catch, thanks!